et/configs.ts

```diff
@@ -142,10 +142,10 @@ export function getConfigSheetName(region: Region, configSheet: keyof (ConfigShe
  */
 export function readInCurrentConfig() {
   const builder = (envVar: string, region: Region) => {
-    return sheets.reduce((acc, sheetName) => {
```

Comment:
Is this change mandated by eslint? I don't particularly prefer <> over "as". Is it more standard?

Reply:
Yeah ESLint did this. It's better because its a more strict type supposedly. Using "as" is frowned upon (although I think I disabled that rule in the end!)